command/cli.go

```diff
@@ -129,14 +135,16 @@ type FixArgs struct {
 
 func (va *ValidateArgs) AddFlagSets(flags *flag.FlagSet) {
 	flags.BoolVar(&va.SyntaxOnly, "syntax-only", false, "check syntax only")
+	flags.BoolVar(&va.MetaArgs.WarnOnUndeclaredVar, "", true, "") //WarnOnUndeclaredVar is true for validate but disabled by no-warn-undeclared-var flag
```

Comment:
I wonder, do we need this? Is this just for documentation purposes?

Reply:
My thinking here is that we have to make sure WarnOnUndeclaredVar is always true for validate. So `MetaArgs.WarnOnUndeclaredVar` has to be set to true somewhere. We can either set it to true within the validate command RunContext method or set it here were the MetaArgs are defined. I opted to keep as close to the definition as possible with a comment to explanin the why. Is this confusing? 

I'm open to other suggestions?